tests/test_vault.py

```diff
@@ -0,0 +1,246 @@
+import secrets
+import string
+
+import pytest
+from lamin_vault.client._create_vault_client import (
+    create_vault_admin_client,
+    create_vault_authenticated_client,
+)
+from lamin_vault.client.postgres._connection_config_db_exists import (
+    connection_config_db_exists,
+)
+from lamin_vault.client.postgres._role_and_policy_exist import role_and_policy_exist
+from lamin_vault.utils._lamin_dsn import LaminDsn
+from lamindb_setup._load_instance import load
+from lamindb_setup._init_instance import init
+from lamindb_setup._init_vault import init_vault
+from lamindb_setup._settings import settings
+from psycopg2 import sql
+from psycopg2.extensions import ISOLATION_LEVEL_AUTOCOMMIT
+from sqlalchemy import create_engine
```

Comment:
Introducing sqlalchemy just for this seems a bit much after we worked so hard to eliminate it from the dependencies :)

Why not just use psycopg2?

Reply:
Yes you're right

Reply:
Thanks, Fred!